NICE.Identity.Authentication.Sdk/AccountController.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace NICE.Identity.Authentication.Sdk
+{
+	public class AccountController : Controller
```

Comment:
AccountController is probably not the correct name for this - we don't really want to expose controllers in the SDK, so I think this should ideally be 'AccountService' which implements IAuthenticationService with the methods Login() and Logout(), and passing the HTTPContext as a parameter in these methods.

Happy to stick with this approach though if it's easier